torch/_dynamo/guards.py

```diff
@@ -41,7 +41,9 @@
 from typing import Any, NamedTuple, NoReturn, Optional, TYPE_CHECKING, Union
 
 
+# TODO: consolidate on 3.11
 try:
+    # pyrefly: ignore [missing-module-attribute]
     from typing import LiteralString
 except ImportError:
     from typing_extensions import LiteralString
```

Comment:
```suggestion
from typing_extensions import LiteralString
```
It's a literally a type alias in newer versions. Also, the proper way to do this is with a Python version guard (if python.version) rather than an ImportError